### ISSUE TITLE
`brotli` feature without `gzip` fails to compile

### ISSUE THREAD

**Issue by kylebarron** [jorgecarleitao/parquet2#196]
I hit this with `v0.14.2` because that's what `arrow2` latest is using:

```
   Compiling parquet2 v0.14.2
error[E0433]: failed to resolve: use of undeclared crate or module `flate2`
   --> /Users/kyle/.cargo/registry/src/github.com-1ecc6299db9ec823/parquet2-0.14.2/src/parquet_bridge.rs:187:28
    |
187 | impl From<BrotliLevel> for flate2::Compression {
    |                            ^^^^^^ use of undeclared crate or module `flate2`
```

But it seems this is still an issue on latest `main`. In `Cargo.toml` the `brotli` feature doesn't seem to require `flate2`, but this `impl` is still defined
https://github.com/jorgecarleitao/parquet2/blob/4cbce012cfe8477bf231caf2fd683871262ba0d4/src/parquet_bridge.rs#L186-L191

**Comment by jorgecarleitao** [comment]
Latest arrow2 is using 0.16, right? I released a fix for this in 0.16.3 (latest 0.16)

**Comment by kylebarron** [comment]
Sorry I meant "latest released arrow2", i.e. arrow2 0.13.1 is still pinned to parquet2 0.14: https://github.com/jorgecarleitao/arrow2/blob/350b690e1d2bb97ef4e220398a39866612acb6a0/Cargo.toml#L72

**Comment by jorgecarleitao** [comment]
Oh, I see. Thanks for clarifying! Released a backport for 0.14 (0.14.3).
